torch/cuda/amp/_autocast_test_lists.py

```diff
@@ -0,0 +1,187 @@
+import torch
```

Comment:
Maybe, put this straight in the testing file, since it seems like this is only used in one place? (Or, if you insist on putting it somewhere importable, maybe in `torch/testing/_internal`, where the rest of our test stuff goes?

Reply:
torch/testing/_internal seems ideal, I didn't know that existed.

Reply:
ok done in my local branch, want to make more changes before repushing